README.md

```diff
@@ -0,0 +1,218 @@
+<a name="readme-top"></a>
+<div align="center">
+    <img src="./logo.png">
+</div>
+
+<div align="center">
+
+  <h2><b>World Vision Clinic</b></h2>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 World Vision Clinic ](#-world-vision-clinic-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [Front-end repository link](#front-end-repository-link)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Check Linter](#check-linter)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+
+# 📖 World Vision Clinic <a name="about-project"></a>
+
+This is a full-stack project. In which you can register yourself create a doctor, book an appointment
+with the docotor, see all appiontmnets, delete a doctor(s).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- User can register himself.
+- Add a doctor.
+- Book an appointment.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+### 🚀 Live Demo <a name="live-demo"></a>
+
+- [API on Render](https://appoint-doctor.onrender.com/)
+
+## Front-end repository link
+
+- <a href="https://github.com/MudasirSherwani/Book-Appointment-Front-End" target="_blank">React Front-end</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+
+ - Laptop/Desktop.
+ - Internet.
+ - Ruby.
+ - Rails.
+ - PostgreSQL.
+ - Vs Code.
+
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+ - mkdir my-folder
+ - cd my-folder
+ - git clone git@github.com:MudasirSherwani/Book-an-Appointment.git
+ - cd Book-an-Appointment
+ - bundle install
+ - db:create
+ - db:migrate
```

Comment:
- [ ] These commands are not working because it's `rails db:create` and `rails db:migrate`. 
Could you correct them on your readme? 👍 

- [ ] Also, if I type `rails db:create` it won't work for new users because the `secret_key_base` is not being pushed to the repo:

![image](https://github.com/MudasirSherwani/Book-an-Appointment/assets/57421823/e2b91cd0-2eaf-49b9-9128-976d6a1765ef)

It's on the `.gitignore`:

![image](https://github.com/MudasirSherwani/Book-an-Appointment/assets/57421823/c3fcf746-469d-4664-a63f-1fdd7924ac23)



## Tips and extra help

This is the correct way to generate a new `master.key`:

```
1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. This command will create a new master.key and credentials.yml.enc if they do not exist.
```
Please, kindly add these instructions to your readme.  That way, your new users will be able to create the database and run the project correctly. 💪 

### Result:

New master.key created:

![image](https://github.com/MudasirSherwani/Book-an-Appointment/assets/57421823/ce272a9a-94e4-415f-b156-346eaa5a89b3)

Database creation now works:

![image](https://github.com/MudasirSherwani/Book-an-Appointment/assets/57421823/5346c3f0-a88a-4684-b6c6-fdd4a0263417)